webpack/config.dev.js

```diff
@@ -0,0 +1,14 @@
+var webpack = require('webpack');
+var merge = require('lodash.merge');
+var baseConfig = require('./config.base');
+
+module.exports = merge(baseConfig, {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client'
+  ].concat(baseConfig.entry),
```

Comment:
Actually, it seems to merge them in-place, by index. I'm guessing this will have unintended side effects as well.